chromadb/errors.py

```diff
@@ -32,6 +32,13 @@ def name(cls) -> str:
         return "InvalidCollection"
 
 
+class CollectionAlreadyExistsError(ChromaError):
```

Comment:
@bhashithe-air, this is great but can we make things a bit more RESTful-ish by adding the response code. Here is an example:

```python
class CollectionNotFoundError(ChromaError):
    @overrides
    def code(self) -> int:
        return 409  # Conflict with current state

    @classmethod
    @overrides
    def name(cls) -> str:
        return "CollectionNotFound"
```

Reply:
Yes let me add this part in as well